social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
The other URI is actually an URI on the prodder's OAuth service, e.g. on google.com etc.  I just didn't want the developer to get confused about which URI is configurable.

Thinking through this some more, the callback URI in our framework specification is rarely changed through configuration.  There really is no need to.  So I'm happy to change from `callbackUri` -> `uri` for consistency, and use the YAML comments to indicate what this is for.

Regarding the need for a different callback uri for each provider, any comments about my previous response?